commands/ci/debug.js

```diff
@@ -76,13 +75,7 @@ function* run (context, heroku) {
     showStatus: false
   })
 
-  let dynoPromise
-  if (attachURL) {
-    dyno.dyno = { attach_url: attachURL }
-    dynoPromise = dyno.attach()
-  } else {
-    dynoPromise = dyno.start()
-  }
+  dyno.dyno = { attach_url: Utils.dig(testNodes, 0, 'dyno', 'attach_url') }
```

Comment:
@gudmundur in that case, I believe `dyno.attach` should return an exception in

https://github.com/heroku/heroku-ci/blob/heroku-run-upstream/commands/ci/debug.js#L94-L95

```js
> Utils.dig([], 0, 'dyno', 'attach_url')
undefined

> {}.dyno = { testNodes: undefined }
 Object {}
```

`Dyno.attach` would fail on `https://github.com/heroku/heroku-run/blob/b5cab4e49fb30b718834ada4fbf14c35d2a30f5d/lib/dyno.js#L95`